subgraph.template.yaml

```diff
@@ -2,6 +2,74 @@ specVersion: 0.0.2
 schema:
   file: ./schema.graphql
 dataSources:
+  - name: Reserve
```

Comment:
This is interesting...

The deployed contracts 'reserve' is actually a ReserveProxy.  The ReserveProxy links to a 'strategy', which is itself a reserve.

It's probably simplest right now to leave the 'reserve' as an address and let the client pull in the recipient and rate directly.